App crashes when clicking the item "CoordinatorLayout Example (3/3)"

```
E/AndroidRuntime: FATAL EXCEPTION: main
  Process: com.google.androidstudio.motionlayoutexample, PID: 13932
  java.lang.RuntimeException: Unable to start activity ComponentInfo{com.google.androidstudio.motionlayoutexample/com.google.androidstudio.motionlayoutexample.DemoActivity}: java.lang.IllegalStateException: findViewById(R.id.motionLayout) must not be null
      at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2778)
      at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856)
      at android.app.ActivityThread.-wrap11(Unknown Source:0)
      at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1589)
      at android.os.Handler.dispatchMessage(Handler.java:106)
      at android.os.Looper.loop(Looper.java:164)
      at android.app.ActivityThread.main(ActivityThread.java:6501)
      at java.lang.reflect.Method.invoke(Native Method)
      at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
   Caused by: java.lang.IllegalStateException: findViewById(R.id.motionLayout) must not be null
      at com.google.androidstudio.motionlayoutexample.DemoActivity.onCreate(DemoActivity.kt:35)
      at android.app.Activity.performCreate(Activity.java:7036)
      at android.app.Activity.performCreate(Activity.java:7027)
      at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1215)
      at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2731)
```

--- a/motionlayout/src/main/res/layout/motion_11_coordinatorlayout.xml
+++ b/motionlayout/src/main/res/layout/motion_11_coordinatorlayout.xml
@@ -15,7 +15,7 @@
   -->
 <android.support.design.widget.CoordinatorLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/content"
+    android:id="@+id/motionLayout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:fitsSystemWindows="false"

